auslib/admin/views/forms.py

```diff
@@ -78,6 +80,34 @@ def coercer(value):
     return coercer
 
 
+def operator(check_version=False):
+    log.debug('starting in operator, check_version is %s' % check_version)
+
+    def _validator(form, field):
+        log.debug('starting in _validator: field.data is %s' % field.data)
+        log.debug('starting in _validator, check_version is %s' % check_version)
+        # empty input is fine
+        if field.data is None:
+            return
+        try:
+            op, operand = get_op(field.data)
+            log.debug('Got (%s, %s) from get_op', op, operand)
+        except TypeError:
+            # get_op field returns None if no operator or no match, can't be unpacked
+            raise ValidationError('Invalid input for %s' % field.label)
+
+        if check_version:
```

Comment:
Have you tested this within the UI? I mentioned this in https://bugzilla.mozilla.org/show_bug.cgi?id=1021022#c8 as well...I'm a bit concerned that input from the actual UI may not end up as Python None. If this works fine within the UI, this is OK.